models/Asset/Image/Thumbnail/Processor.php

```diff
@@ -250,6 +257,7 @@ public static function process(Asset $asset, Config $config, $fileSystemPath = n
                 $fileExists = true;
             }
         } catch (\Exception $e) {
+            Logger::warn($e->getMessage());
```

Comment:
@dvesh3 I am not sure if we should add logging here. We should may get rid of the try and catch. What do you think?

Reply:
we should still handle cases when there's `FilesystemException` from flysystem.

Reply:
Then we can maybe ignore the exception when the file doesn't exist

Reply:
how about we change the log level to debug? we do this for thumbnail generation as well.

Reply:
```suggestion
            Logger::debug($e->getMessage());
```